fec/fec/settings/base.py

```
@@ -235,3 +235,18 @@
     DEFAULT_FILE_STORAGE = 'storages.backends.s3boto3.S3Boto3Storage'
     AWS_LOCATION = 'cms-content'
     AWS_S3_REGION_NAME = env.get_credential('CMS_AWS_DEFAULT_REGION')
+
+UAA_CLIENT_ID = env.get_credential('CMS_LOGIN_CLIENT_ID', 'my-client-id')
+UAA_CLIENT_SECRET = env.get_credential('CMS_LOGIN_CLIENT_SECRET', 'my-client-secret')
+#fake uaa server deploys locally on port 8080.  Will be needed to login for local use
+#TODO: These will have to have a explicit reference until we can figure out how
```

@ccostino Wanted to let you know I made this change, and to solicit your approval here.  In short, unless we can figure out how to silence the error in the tests, the urls for these variables have to be `https`, or the test errors out.  Maybe we can revisit later but for now I say we keep them as is, and add as environment variables later on in a pull request?